src/plugins/kibana_utils/public/index.ts

```diff
@@ -18,18 +18,19 @@
  */
 
 export {
+  calculateObjectHash,
```

Comment:
calculateObjectHash wasn't exported at the top level so I added it here, and alphabetized